contentcuration/contentcuration/frontend/channelEdit/views/ImportFromChannels/ImportFromChannelsModal.vue

```diff
@@ -209,20 +209,23 @@
         });
       },
       handleClickImport() {
-        const nodeIds = this.selected.map(({ id }) => id);
-        return this.copyContentNodes({
-          id__in: nodeIds,
-          target: this.$route.params.destNodeId,
-        }).then(() => {
-          // When exiting, do not show snackbar when clearing selections
-          this.showSnackbar = false;
-          this.$store.commit('importFromChannels/CLEAR_NODES');
-          this.$router.push({
-            name: RouteNames.TREE_VIEW,
-            params: {
-              nodeId: this.$route.params.destNodeId,
-            },
-          });
+        Promise.allSettled(
+          this.selected.map(({ id }) => {
+            return this.copyContentNode({
+              id,
+              target: this.$route.params.destNodeId,
+              wait_for_status: true,
+            });
+          })
+        );
```

Comment:
Using `Promise.allSettled` so that the rejections from the underlying promise doesn't result in console errors. Because here we don't need to handle rejected promises from `copyContentNode`. Does it make sense sir?

Reply:
Should I use an empty catch here? (maybe a better alternative)